github-wide.css

```diff
@@ -120,3 +120,8 @@ button.discussion-sidebar-toggle {
 .vcard-avatar {
   width: 230px !important;
 }
+
+/* Commits: extended message under "..." */
+.commit-desc pre {
+  max-width: inherit;
```

Comment:
It inherits the `div` wrapper's max-width. Although I wanted just to unset it, so `none` might be better.